.vscode/settings.json

```diff
@@ -24,5 +24,15 @@
       },
       "Prettier-SQL.expressionWidth": 80,
       "Prettier-SQL.SQLFlavourOverride": "postgresql",
-      "go.lintTool": "golangci-lint-v2"
+      "mise.configureExtensionsUseSymLinks": true,
+      "go.lintTool": "golangci-lint-v2",
+      "go.goroot": "${workspaceFolder}/.vscode/mise-tools/goRoot",
+      "debug.javascript.defaultRuntimeExecutable": {
```

Comment:
Let's not have very personal items in the `settings.json`, I have custom paths (and custom builds) of some tools that would be ignored (and therefore broken) by those. Not to mention a setting for NodeJS in the Go project.

Reply:
In my opinion, the best approach is to disable the automatic configuration of extensions ([see extension documentation](https://hverlin.github.io/mise-vscode/tutorials/settinguptheextension/#automatic-configuration-of-vs-code-extensions)).

The automatic configuration is a bit cumbersome, as the extension attempts to modify the workspace settings when we manually trigger the auto-config.

I have moved all related configuration into a user settings sample file to keep the “Getting Started” process simple, and I have updated the documentation accordingly.